bugshaker/src/main/java/com/github/stkent/bugshaker/utilities/SharedPreferencesUtil.java

```diff
@@ -0,0 +1,51 @@
+package com.github.stkent.bugshaker.utilities;
+
+import java.util.Set;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+public class SharedPreferencesUtil {
+
+	private static final String EMAIL_ADDRESS = "emailAddress";
+	private static final String EMAIL_ADDRESS2 = "emailAddress2";
```

Comment:
they're just badly named, one of them is actually for the email subject line :o so they have been renamed